test/.gitignore

```diff
@@ -0,0 +1,2 @@
+# dependencies
+/node_modules
```

Comment:
Where is the most appropriate place for this kind of ignore? Is there any place in the project you would want to check node_modules into source?